lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -375,6 +393,33 @@ class LighthouseReportViewer {
       self.opener.postMessage({opened: true}, '*');
     }
   }
+
+  /**
+   * @param {PSIParams} params
+   */
+  _loadFromPSI(params) {
+    const loadingOverlayEl = document.createElement('div');
+    loadingOverlayEl.classList.add('lh-loading-overlay');
+    loadingOverlayEl.textContent = 'Waiting for Lighthouse results ...';
+    find('.viewer-placeholder-inner', document.body).classList.add('lh-loading');
+    document.body.appendChild(loadingOverlayEl);
```

Comment:
can use the `logger` infrastructure (`logger.log`, I guess)

Reply:
do you mean instead of showing an overlay?

Reply:
> do you mean instead of showing an overlay?

yes (restyling from 2016-toast-style if needed)

Reply:
oh i didnt realize that was a toasty thing